src/mmw/js/src/modeling/views.js

```diff
@@ -275,7 +276,8 @@ var ScenarioTabPanelView = Marionette.ItemView.extend({
     templateHelpers: function() {
         return {
             cid: this.model.cid,
-            editable: isEditable(this.model)
+            editable: isEditable(this.model),
+            isNew: this.model.isNew()
```

Comment:
Since Nunjucks templates follow Python conventions, consider naming this variable `is_new`